eox_nelp/openedx_filters/xapi/filters.py

```diff
@@ -107,17 +107,21 @@ def run_filter(self, transformer, result):  # pylint: disable=arguments-differ,
         }
 
 
-class XApiBaseProblemsFilter(PipelineStep):
-    """This filter is designed to modify object attributes of an event which transformer class
-    is a child of BaseProblemsTransformer, this will add the description field and will change
-    the name based on the course language.
+class XApiModuleQuestionObjectFilter(PipelineStep):
+    """This filter is designed to modify object attributes of events whose object type is a module
+    or a question, this will add the description field and will change the name based on the course language.
 
     How to set:
         OPEN_EDX_FILTERS_CONFIG = {
             "event_routing_backends.processors.xapi.problem_interaction_events.base_problems.get_object": {
-                "pipeline": ["eox_nelp.openedx_filters.xapi.filters.XApiBaseProblemsFilter"],
+                "pipeline": ["eox_nelp.openedx_filters.xapi.filters.XApiModuleQuestionObjectFilter"],
```

Comment:
I like more the name related `problems`, why you change it to `questions`?

Reply:
As I mentioned in the doc-string this filter works with events that have a module or question object type, so this is not a unique filter for problems, and why question because a  `edx.grades.problem.submitted` event has a question definition type https://github.com/nelc/event-routing-backends/blob/master/event_routing_backends/processors/xapi/event_transformers/problem_interaction_events.py#L40